packages/constellation/src/components/Primitives/Modal/ModalFooter.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import { Button } from '../../Compositions'
+import { ModalFooterProps } from './Modal.types'
+
+/**
+ * The ModalFooter can be passed to the Modal component and serves as the main location for user actions.
+ */
+
+const ModalFooter = ({ primaryCta, secondaryCta }: ModalFooterProps) => (
```

Comment:
Maybe would make more sense if the ModalFooter just accepts an array of Buttons?

Then, if we need 3 buttons in the footer, we can pass it and not have to change this component. Would that work, or is this other issue that would prevent that?

Reply:
@awiermanncasas-bc yea that's definitely something I considered. My only concern was trying to avoid situations where someone potentially added two or more "primary" buttons. For modals, IMO it only makes sense to have a single primary action (and possibly a secondary action).

As an alternative, The modal can be passed an open-ended set of children. So if the consumer needs to render a larger set of buttons it can also be done through passing them as children.